pkg/api/api.go

```diff
@@ -339,6 +339,8 @@ func (hs *HTTPServer) registerRoutes() {
 		apiRoute.Any("/plugins/:pluginId/resources", authorize(reqSignedIn, ac.EvalPermission(plugins.ActionAppAccess, pluginIDScope)), hs.CallResource)
 		apiRoute.Any("/plugins/:pluginId/resources/*", authorize(reqSignedIn, ac.EvalPermission(plugins.ActionAppAccess, pluginIDScope)), hs.CallResource)
 		apiRoute.Get("/plugins/errors", routing.Wrap(hs.GetPluginErrorsList))
+		apiRoute.Any("/plugin-proxy/:pluginId/*", authorize(reqSignedIn, ac.EvalPermission(plugins.ActionAppAccess, pluginIDScope)), hs.ProxyPluginRequest)
+		apiRoute.Any("/plugin-proxy/:pluginId", authorize(reqSignedIn, ac.EvalPermission(plugins.ActionAppAccess, pluginIDScope)), hs.ProxyPluginRequest)
```

Comment:
Just confirming we support requests at the root - do we have examples of this where route.Path is set to `""` or `"/"`?

Reply:
Haven't found any, but in theory we could support it. Same as datasource proxy. Do you see any issue with that?

Reply:
I don't think so... Have just checked against GCOM in any case for curiosity and don't think it exists. Can keep or omit I reckon

Reply:
I'll suggest we stick with this just in case